x/concentrated-liquidity/types/keys.go

```diff
@@ -80,6 +80,11 @@ func KeyPosition(poolId uint64, addr sdk.AccAddress, lowerTick, upperTick int64)
 	return []byte(fmt.Sprintf("%s%s%s%s%d%s%d%s%d", PositionPrefix, KeySeparator, addrKey, KeySeparator, poolId, KeySeparator, lowerTick, KeySeparator, upperTick))
 }
 
+func KeyPoolIdAndAddress(poolId uint64, addr sdk.AccAddress) []byte {
```

Comment:
nit: order of params should probably be the same as the order they are keyed in (i.e. `addr, poolId` instead of `poolId, addr`)

Reply:
https://github.com/osmosis-labs/osmosis/pull/4538/commits/e4df839415a3f5447a71ab4f201adc4598c24d7e